node/test/bundle.test.mjs

```diff
@@ -414,4 +414,29 @@ test('should support throwing in visitors', async () => {
   assert.equal(error.message, 'Some error');
 });
 
+test('external import', async () => {
+  const { code: buffer } = await bundleAsync(/** @type {import('../index').BundleAsyncOptions} */ ({
+    filename: 'tests/testdata/has_external.css',
+    resolver: {
+      resolve(specifier, originatingFile) {
+        if (specifier === './does_not_exist.css' || specifier.startsWith('https:')) {
+          return true;
```

Comment:
For now, I made `resolve` to accept `string | true` to focus on the rough design. I think `string | { specifier: string, external?: boolean }` would be nice here. What do you think?